libmscore/chord.cpp

```diff
@@ -700,6 +700,9 @@ void Chord::addLedgerLines()
       qreal lineDistance = 1;
       qreal mag         = 1;
       bool staffVisible  = true;
+      // for staff changes
+      qreal yoffset = 0.0;
```

Comment:
I might have missed something in the Telegram dicussion, but what it the reason for not using `yOffset`?

Reply:
Yes, that would be the reason.(Not sure if a valid one tho).
There are a few  `yoffset` in the code that don't follow capitalization convention, `staffType()->yoffset()` among them.